### DISCUSSION TITLE
Rename Project: Name Suggestions.

### DISCUSSION THREAD

**Discussion by BotBlake** [BotBlake/pyTAB#33] [General]
When I started with pyTAB, the name of the project was not relevant - admittedly, it still isn't - BUT now that the first alpha release is at least within reach, a name should be found that is a bit more fitting and not already in use.

pyTAB is not a suitable name here, as many other projects are already using it.

I would like to ask you for some name suggestions to rename the project.

Kind regards
BotBlake

**Comment by brys0** [comment]
You can do something more unique, but considering this benchmarking app is being written in at least one other language, (my bad) it should probably follow a naming convention that's easy to follow.

jelly - the jelly part of jellyfin, typically a prefix to many jellyfin based applications

bench - describes the use of the application as a benchmarking tool related to jellyfin

-py - append it with the language used for the application. 

jellybench-py 
jellybench-go
jellybench-csharp
Etc..
